docs/REST API/rest-api.md

```diff
@@ -1820,7 +1820,8 @@ None.
 - [Get mobile device management (MDM) summary](#get-mobile-device-management-mdm-summary)
 - [Get host's macadmin mobile device management (MDM) and Munki information](#get-hosts-macadmin-mobile-device-management-mdm-and-munki-information)
 - [Get aggregated host's mobile device management (MDM) and Munki information](#get-aggregated-hosts-macadmin-mobile-device-management-mdm-and-munki-information)
-- [Get host OS versions](#get-host-os-versions)
+- [List host OS versions](#list-host-os-versions)
+- [Get host OS version](#get-host-os-version)
```

Comment:
@rachaelshaw seems like we're replacing `get`, should this line be removed?

Reply:
Before I delete a bunch of stuff, just want to double-check: we only ended up using `GET /api/v1/fleet/os_versions` and didn't need to add an endpoint for `GET /api/v1/fleet/os_versions/:id`?

Reply:
Correct, we can use the same endpoint with a `name` and `version` filter to get the information on the osDetails page.  Also since we are [aggregating](https://github.com/fleetdm/fleet/issues/16234) operating systems, there's not a single id per operating system.